phoenix5-spark/src/it/scala/org/apache/phoenix/spark/PhoenixSparkSqlIT.scala

```diff
@@ -0,0 +1,95 @@
+package org.apache.phoenix.spark
+
+import org.apache.phoenix.spark.datasource.v2.PhoenixDataSource
+import org.apache.spark.sql.Row
+
+import scala.collection.mutable.ListBuffer
+
+/**
+ * Note: If running directly from an IDE, these are the recommended VM parameters:
+ * -Xmx1536m -XX:MaxPermSize=512m -XX:ReservedCodeCacheSize=512m
+ */
+class PhoenixSparkSqlIT extends AbstractPhoenixSparkIT {
+  val sqlTableName = "SQL_TABLE"
+
+  after {
+    spark.sql(s"DROP TABLE IF EXISTS  $sqlTableName")
+  }
+
+  test("Can read from table using spark-sql") {
+    val expected : Array[Row] = Array(
+      Row.fromSeq(Seq(1, "test_row_1")),
+      Row.fromSeq(Seq(2, "test_row_2"))
+    )
+
+    spark.sql(s"CREATE TABLE $sqlTableName USING phoenix " +
+      s"OPTIONS ('table' 'TABLE1', '${PhoenixDataSource.JDBC_URL}' '$jdbcUrl')")
+
+    val dataFrame = spark.sql(s"SELECT * FROM $sqlTableName")
+
+    dataFrame.show()
+
+    dataFrame.collect() shouldEqual expected
+  }
+
+  test("Can read from table using spark-sql with where clause and selecting specific columns`") {
+    val expected : Array[Row] = Array(
+      Row.fromSeq(Seq("test_row_1"))
+    )
+
+    spark.sql(s"CREATE TABLE $sqlTableName USING phoenix " +
+      s"OPTIONS ('table' 'TABLE1', '${PhoenixDataSource.JDBC_URL}' '$jdbcUrl')")
+
+    val dataFrame = spark.sql(s"SELECT COL1 as LABEL FROM $sqlTableName where ID=1")
+    dataFrame.show
```

Comment:
While these may be useful for debugging, In general we shouldn't print to the stdout in tests.

Reply:
Forgot removing this before push. Done now.